shared/chat/inbox/row/add-new-row.js

```diff
@@ -10,87 +10,110 @@ if (!isMobile) {
   KeyHandler = require('../../../util/key-handler.desktop').default
 }
 
-const _AddNewRow = ({
-  onNewChat,
-  onJumpToChat,
-  isLoading,
-}: {
+type Props = {
   onNewChat: () => void,
+  onSetFilter: (filter: string) => void,
   onJumpToChat: () => void,
   isLoading: boolean,
-}) => (
-  <Box style={{...globalStyles.flexBoxRow, alignItems: 'center', minHeight: 48, position: 'relative'}}>
-    <ClickableBox
-      style={{
-        ...globalStyles.flexBoxRow,
-        alignItems: 'center',
-        flexGrow: 1,
-        flexShrink: 0,
-        justifyContent: 'center',
-        paddingLeft: globalMargins.medium,
-        paddingRight: globalMargins.medium,
-      }}
-      onClick={onNewChat}
-    >
-      <Icon type="iconfont-new" style={{color: globalColors.blue, marginRight: 9}} />
-      <Text type="BodyBigLink">New chat</Text>
-    </ClickableBox>
-    {isLoading &&
-      <Box style={{bottom: 0, left: 0, position: 'absolute', right: 0}}>
-        <LoadingLine />
-      </Box>}
-  </Box>
-)
+}
+
+type State = {
+  inEditingMode: boolean,
+  filter: string,
+}
+
+class _AddNewRow extends Component<void, Props, State> {
```

Comment:
Should this class be renamed? It does more than add a new chat now, and "AddNewRow" sounds like a command. "ChatFilterRow" perhaps?